media-queries.css

```diff
@@ -0,0 +1,115 @@
+@media screen and (max-width: 991px) {
+
+    h3 {
```

Comment:
in mobile view you need to reduce the navbar size, and you need to look into the inline padding. all other issues will solve once you do flex properly.